src/view/screens/Search/Search.tsx

```diff
@@ -316,8 +324,33 @@ function SearchScreenUserResults({query}: {query: string}) {
   )
 }
 
-const SECTIONS_LOGGEDOUT = ['Users']
-const SECTIONS_LOGGEDIN = ['Posts', 'Users']
+const SECTIONS_LOGGEDOUT = ['PEOPLE'] as const
+const SECTIONS_LOGGEDIN = ['TOP', 'LATEST', 'PEOPLE'] as const
+const SECTIONS_LOGGEDOUT_LEGACY = ['USERS'] as const
+const SECTIONS_LOGGEDIN_LEGACY = ['POSTS', 'USERS'] as const
+
+function getSectionName(
+  i18n: I18n,
+  section:
+    | (typeof SECTIONS_LOGGEDIN)[number]
+    | (typeof SECTIONS_LOGGEDOUT)[number]
+    | (typeof SECTIONS_LOGGEDIN_LEGACY)[number]
+    | (typeof SECTIONS_LOGGEDOUT_LEGACY)[number],
+) {
+  switch (section) {
+    case 'TOP':
+      return i18n._(msg`Top`)
```

Comment:
I think (?) we shouldn’t use the global `_` because it won’t update on lang change. https://lingui.dev/tutorials/react-patterns

Let’s just move this inline into the component. Maybe right into the useMemo that wants it.